From: Hanjun Guo <guohanjun@huawei.com>
Date: Thu, 24 Sep 2020 10:56:53 +0800
Subject: ACPI: cmos_rtc: Remove leftover ACPI_MODULE_NAME()
Patch-mainline: v5.10-rc1
Git-commit: 1eb3d0414520c8ee91b78b9f24cf30ffe71b6e5d
References: jsc#SLE-16407

ACPI_MODULE_NAME() is only needed for ACPICA debug functionality
such as ACPI_DEBUG_PRINT() which is not used in acpi_cmos_rtc.c,
remove it.

Signed-off-by: Hanjun Guo <guohanjun@huawei.com>
[ rjw: Subject edit ]
Signed-off-by: Rafael J. Wysocki <rafael.j.wysocki@intel.com>
Acked-by: Lee, Chun-Yi <jlee@suse.com>
---
 drivers/acpi/acpi_cmos_rtc.c |    2 --
 1 file changed, 2 deletions(-)

--- a/drivers/acpi/acpi_cmos_rtc.c
+++ b/drivers/acpi/acpi_cmos_rtc.c
@@ -15,8 +15,6 @@
 
 #include "internal.h"
 
-ACPI_MODULE_NAME("cmos rtc");
-
 static const struct acpi_device_id acpi_cmos_rtc_ids[] = {
 	{ "PNP0B00" },
 	{ "PNP0B01" },
